src/frontend/ai-reviewer-admin/src/domain/documents/_tests_/DocumentTypeEditor.test.js

```diff
@@ -25,10 +24,10 @@ describe("DocumentTypeEditor test suite", () => {
     service.getDocumentTypeConfigurations = jest.fn(() =>
       Promise.resolve(configurations)
     );
-    const { getByText } = render(<DocumentTypeEditor />);
+    const { getAllByText } = render(<DocumentTypeEditor />);
     await waitFor(() => {});
 
-    const sampleData = getByText("Response to Civil Claim");
+    const sampleData = getAllByText("Response to Civil Claim")[0];
     expect(sampleData).toBeInTheDocument();
```

Comment:
I see why you're doing this, getByText errors when it can't find the node or if there is more than one.
But this assert is checking if there is at least one.
Maybe rather you could get the entire array (instead of pulling off the first record [0]) and assert the size of the array is 2 (since you added a second documentType with the same description), "Response to Civil Claim". There should be RCC and DEVRCC in the list.

Reply:
good catch makes sense :+1: